clients/docs/includes/configs/cloud/restproxy-sr.config

```diff
@@ -0,0 +1,15 @@
+# Required connection configs for Kafka producer, consumer, and admin
+bootstrap.servers={{ BROKER_ENDPOINT }}
+security.protocol=SASL_SSL
+sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="{{ CLUSTER_API_KEY }}" password="{{ CLUSTER_API_SECRET }}";
+sasl.mechanism=PLAIN
+client.bootstrap.servers={{ BROKER_ENDPOINT }}
+client.security.protocol=SASL_SSL
+client.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="{{ CLUSTER_API_KEY }}" password="{{ CLUSTER_API_SECRET }}";
+client.ssl.endpoint.identification.algorithm=https
```

Comment:
@cryoshida is this a required config, or is the default already like the Java client and can it be removed?

Reply:
it's a required config, from [docs](https://docs.confluent.io/platform/current/kafka-rest/production-deployment/rest-proxy/config.html):
```
client.ssl.endpoint.identification.algorithm
The endpoint identification algorithm to validate server hostname using server certificate.

Type: string
Default: null
Importance: low
```

Reply:
@cryoshida I don't think it's required as evidenced by this RP config that works with CCloud: 

https://github.com/confluentinc/cp-all-in-one/blob/7.0.1-post/cp-all-in-one-cloud/docker-compose.yml#L146-L165

Note: https://github.com/confluentinc/cp-all-in-one/pull/40

Reply:
sure thing! they're removed now.